src/run.ts

```diff
@@ -0,0 +1,121 @@
+import * as core from '@actions/core';
+import { load } from "js-yaml";
+import { readFileSync } from "fs";
+import { throttling } from '@octokit/plugin-throttling';
+import { GitHub, getOctokitOptions } from '@actions/github/lib/utils'
+type Octokit = InstanceType<typeof GitHub>;
+
+interface Input {
+  token: string;
+  org: string;
+  file: string;
+  forceEnable: boolean;
+}
+
+export function getInputs(): Input {
+  const result = {} as Input;
+  result.token = core.getInput('github-token');
+  result.org = core.getInput('org');
+  result.file = core.getInput('file');
+  result.forceEnable = core.getBooleanInput('force-enable');
+  return result;
+}
+
+const getRepoNames = async (octokit: Octokit, orgLogin: string): Promise<string[]> => {
+  let repoNames: string[] = [];
+  let _hasNextPage = true;
+  let _endCursor = null;
+  while (_hasNextPage) {
+    const {
+      organization: {
+        repositories: {
+          nodes: repositories,
+          pageInfo: {
+            hasNextPage,
+            endCursor
+          }
+        }
+      }
+    } = await octokit.graphql(`{ 
```

Comment:
I think you can be a bit more strict here with your typing. The `octokit.graphql` will return type `any`, which isn't great. For your graphql calls, I would be clearer with your typing. 

You don't have to map out the entire response, but I would expect:

```
      organization: {
        repositories: {
          nodes: repositories,
          pageInfo: {
            hasNextPage,
            endCursor
          }
        }
      }
```

To be typed, especially `repositories` as you are using that later in your `.map()` and `.filter()`

Reply:
I added the type `import { Organization } from "@octokit/graphql-schema";`